source/adios2/toolkit/sst/dp/rdma_dp.c

```diff
@@ -692,7 +692,8 @@ static void RdmaProvideTimestep(CP_Services Svcs, DP_WS_Stream Stream_v,
     sprintf(Info->CheckString, "Rdma info for timestep %ld from rank %d",
             Timestep, Stream->Rank);
     Info->CheckInt = Stream->Rank * 1000 + Timestep;
-    Entry->Data = Data;
+    Entry->Data = malloc(sizeof(*Data));
```

Comment:
Is this ever freed?

Reply:
No, it is not. I am getting a commit ready adding cleanup/deallocation for the DP, but I wanted to start a PR for this since the RDMA DP is currently broken without the above change.

Reply:
Now it is